Mage/src/main/java/mage/abilities/effects/common/ReturnSourceFromGraveyardToBattlefieldWithCounterEffect.java

```diff
@@ -20,7 +21,11 @@ public class ReturnSourceFromGraveyardToBattlefieldWithCounterEffect extends Ret
     public ReturnSourceFromGraveyardToBattlefieldWithCounterEffect(Counter counter, boolean tapped) {
```

Comment:
While you're editing this, could you change this to call your new constructor (rather than super) so that the defaults  are clear

Reply:
Done